daemon/lightning/lnd/lnd.go

```diff
@@ -352,17 +351,28 @@ func (dc *Client) GenerateInvoice(ctx context.Context, amountSats decimal.Decima
 		DescriptionHash: nil,                         // Optional description hash
 	}
 
-	res, err := dc.lndClient.AddInvoice(ctx, invoiceReq)
+	res, err := c.lndClient.AddInvoice(ctx, invoiceReq)
 	if err != nil {
 		return "", nil, err
 	}
 
 	return res.PaymentRequest, res.RHash, nil
 }
 
+func (c *Client) GenerateAddress(ctx context.Context) (string, error) {
+	res, err := c.lndClient.NewAddress(ctx, &lnrpc.NewAddressRequest{
+		Type: lnrpc.AddressType_UNUSED_WITNESS_PUBKEY_HASH,
```

Comment:
Please add a todo here so we don't forget to parametrize these, maybe a customer wants `P2TR`

Reply:
added